cli/packages/cmd/init.go

```diff
@@ -52,34 +51,48 @@ var initCmd = &cobra.Command{
 
 		httpClient := resty.New()
 		httpClient.SetAuthToken(userCreds.UserCredentials.JTWToken)
-		workspaceResponse, err := api.CallGetAllWorkSpacesUserBelongsTo(httpClient)
+
+		organizationResponse, err := api.CallGetAllOrganizations(httpClient)
 		if err != nil {
-			util.HandleError(err, "Unable to pull projects that belong to you")
+			util.HandleError(err, "Unable to pull organizations that belong to you")
 		}
 
-		workspaces := workspaceResponse.Workspaces
-		if len(workspaces) == 0 {
-			message := fmt.Sprintf("You don't have any projects created in Infisical. You must first create a project at %s", util.INFISICAL_TOKEN_NAME)
-			util.PrintErrorMessageAndExit(message)
+		organizations := organizationResponse.Organizations
+
+		organizationNames := util.GetOrganizationsNameList(organizationResponse)
+
+		prompt := promptui.Select{
+			Label: "Which of your Infisical organization would you like to get projects from?",
```

Comment:
let's change to `Which Infisical organization would you like to select a project from?`